.travis.yml

```diff
@@ -9,9 +9,11 @@ before_install:
 before_script:
   - npm install jpm -g
   - cd ..
-  - wget "ftp://ftp.mozilla.org/pub/firefox/releases/38.0b9/linux-x86_64/en-US/firefox-38.0b9.tar.bz2" -O firefox.tar.bz2
+  - wget "http://ftp.mozilla.org/pub/firefox/releases/39.0/linux-x86_64/en-US/firefox-39.0.tar.bz2" -O firefox.tar.bz2
   - tar -xjf firefox.tar.bz2 firefox
   - cd $TRAVIS_BUILD_DIR
- 
+  - npm install istanbul
```

Comment:
Using a name other than `package.json` doesn't work. I got the following error -- 
```
npm WARN package.json privly@0.5.0 No repository field.
npm ERR! not a package /tmp/npm-2308-7W8Ylprv/1440505038075-0.6826705487910658/tmp.tgz
npm ERR! 404 Not Found
npm ERR! 404 
npm ERR! 404 'npm_package.json' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 It was specified as a dependency of 'privly'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.
```
and I think it's not possible(a similar question was asked [here](https://github.com/npm/npm/issues/8222)).